src/dialogs/DontAskAgainDialog/DontAskAgainDialog.js

```diff
@@ -49,10 +49,16 @@ export const DontAskAgainDialog = ({ id, open, onClose, labels, onConfirm }) =>
       </DialogContent>
       <DialogActions>
         {/* TODO Use button1Props and button2Props to enable variants and colors */}
```

Comment:
This comment is no longer relevant, you added two two props mentioned :slightly_smiling_face:

Reply:
DONE